libs/application/templates/parental-leave/src/lib/directorateOfLabour.utils.ts

```diff
@@ -33,7 +34,7 @@ export const monthsToDays = (months: number) => {
 /**
  * Return true if the dates are the first day and the last day of the same month
  */
-const isFullMonth = (start: Date, end: Date) => {
+const fullMonthDates = (start: Date, end: Date) => {
```

Comment:
The previous name was much more descriptivie ... specified that the returned value was indeed a boolean.

Reply:
The only thing it was missing is that the name implied that this was a property, not a method/function.

Reply:
Sounds fair, not sure why I changed the function name in the middle, will change it back 👍